build.sbt

```diff
@@ -155,7 +155,7 @@ lazy val docs = project("docs")
       "akka.version" -> Dependencies.akkaVersion,
       "scala.binaryVersion" -> scalaBinaryVersion.value,
       "scala.version" -> scalaVersion.value,
-      "scaladoc.version" -> scalaVersion.value,
+      "scaladoc.version" -> (if (isSnapshot.value) crossScalaVersions.value.tail.head else crossScalaVersions.value.head),
```

Comment:
This can lead to broken links if scaladoc is generated with scalaVersion set to 2.11, which for some reason was the case for version 10.0.5. Maybe I am not understanding what this is trying to accomplish.